index.d.ts

```diff
@@ -119,5 +122,234 @@ declare module "hyperapp" {
   function memo(view: View, props: PropList): Partial<VDOM>
 
   // The `text` function creates a virtual DOM node representing plain text.
-  function text(value: string | number, node?: Node): VDOM
+  function text(value: number | string, node?: Node): VDOM
+
+  // ---------------------------------------------------------------------------
+
+  // Due to current limitations with TypeScript (which should get resolved in
+  // the future: https://github.com/microsoft/TypeScript/pull/40336), here is
+  // a collection of modified copies of relevant event maps from TypeScript's
+  // "lib.dom.d.ts" definition file to assist with defining `EventActions`:
+
+  type OnElementEventMap = {
+    "onfullscreenchange": Event
```

Comment:
@icylace Where does `Event` come from?

Reply:
@jorgebucaran It comes from TypeScript itself: https://github.com/microsoft/TypeScript/blob/d89635381a65229c5ab5b0c73de15347b86235a2/lib/lib.dom.d.ts#L5285